the-collab-lab/tcl-27-smart-shopping-list
# New update! User can now logout and styling has been updated!

## Description

* Implemented logout feature so user can logout
* Included a welcome prompt with instructions to the Home component
* Imported react-bootstrap npm package
* Styled Home and AddItem component with bootstrap
* Refactored AddItem component to have accessibility features for inputs

## Related Issue

#13 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |


## Testing Steps / QA Criteria

* From your terminal, pull down this branch with git pull origin dp-sk-logout and check that branch out with git checkout dp-sk-logout
* Install new dependencies with npm install
* Once logged in, check for new styling in the Home and AddItem component.
* Check for accessibility in the AddItem component.
* Click logout and make sure you are directed to the home page and localStorage is cleared.
